--- a/src/Clients/FSpot/ui/main_window.ui
+++ b/src/Clients/FSpot/ui/main_window.ui
@@ -150,7 +150,7 @@
             <property name="label" translatable="yes">Rotate _Left</property>
             <signal handler="HandleRotate270Command" name="activate"/>
           </object>
-          <accelerator key="bracketleft" modifiers=""/>
+          <accelerator key="bracketleft" modifiers="GDK_CONTROL_MASK"/>
         </child>
         <child>
           <object class="GtkAction" id="rotate_right">
@@ -158,7 +158,7 @@
             <property name="label" translatable="yes">Rotate _Right</property>
             <signal handler="HandleRotate90Command" name="activate"/>
           </object>
-          <accelerator key="bracketright" modifiers=""/>
+          <accelerator key="bracketright" modifiers="GDK_CONTROL_MASK"/>
         </child>
         <child>
           <object class="GtkAction" id="sharpen">
@@ -166,7 +166,7 @@
             <property name="label" translatable="yes">_Sharpen...</property>
             <signal handler="HandleSharpen" name="activate"/>
           </object>
-          <accelerator key="s"/>
+          <accelerator key="s" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
         </child>
         <child>
           <object class="GtkAction" id="adjust_time">
@@ -196,7 +196,7 @@
             <property name="label" translatable="yes">_Remove From Catalog</property>
             <signal handler="HandleRemoveCommand" name="activate"/>
           </object>
-          <accelerator key="Delete"/>
+          <accelerator key="Delete" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
         </child>
         <child>
           <object class="GtkAction" id="set_as_background">
@@ -351,7 +351,7 @@
             <property name="label" translatable="yes">_Loupe</property>
             <signal handler="HandleLoupe" name="activate"/>
           </object>
-          <accelerator key="v"/>
+          <accelerator key="v" modifiers="GDK_CONTROL_MASK | GDK_SHIFT_MASK"/>
         </child>
         <child>
           <object class="GtkAction" id="tag_icon_size">
