--- a/external/taglib-sharp/src/taglib-sharp.csproj
+++ b/external/taglib-sharp/src/taglib-sharp.csproj
@@ -12,7 +12,7 @@
     </ApplicationIcon>
     <AssemblyName>taglib-sharp</AssemblyName>
     <ReleaseVersion>2.0.4.0</ReleaseVersion>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <TargetFrameworkProfile />
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
@@ -294,4 +294,4 @@
   <ItemGroup>
     <None Include="TagLib\TagLib.sources" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
